lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -0,0 +1,427 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import * as rtlUtils from '../../../__testutils__/rtlUtils';
+
+import userEvent from '@testing-library/user-event';
+
+import NxCombobox, {Props} from '../NxCombobox';
+
+describe('NxCombobox', function() {
+  const minimalProps: Props<string | number> = {
+        value: '',
+        onChange: () => {},
+        onSearch: () => {},
+        matches: []
+      },
+      rtlRender = rtlUtils.rtlRender(NxCombobox, minimalProps);
+
+  //mock scrollIntoView function using jest
+  window.HTMLElement.prototype.scrollIntoView = jest.fn();
+
+  it('renders a div with the nx-combobox class and the specified attributes', function() {
+    const { container } = rtlRender({ title: 'bar' }),
+        component = container.firstElementChild!;
+
+    expect(component.tagName).toBe('DIV');
```

Comment:
In RTL, I'm of the opinion that we should only check tag names on elements that have a lot of built-in behavior or rules about how they can be nested, which the calling code needs to know about.  `div` is not that case, so I'd say the fact that the wrapping element is a `div` is an implementation detail that shouldn't be tested.

Reply:
Thanks for the clarification, updated here 4029d58f1778f46cf53cdedad783b595e4a9da4e.